### PR TITLE
Improve consistency in code

### PR DESCRIPTION
**Pull request by 8kdesign**

### CODE REVIEW

File: src/main/java/seedu/duke/commands/ListModuleCommand.java [hunk 1]
```diff
@@ -1,23 +1,23 @@
 package seedu.duke.commands;
 
 import seedu.duke.common.Messages;
-import seedu.duke.exceptions.CommandException;
+import seedu.duke.exception.CommandException;
 import seedu.duke.module.ModuleList;
 import seedu.duke.ui.UI;
 
-import static java.util.Objects.requireNonNull;
+import static seedu.duke.common.Messages.MESSAGE_MODULE_TO_LIST;
```
Comment (aliciatay-zls) [inline src/main/java/seedu/duke/commands/ListModuleCommand.java:8]: Actually should this class be called ListModule**s**Command? To standardise with ListLesson**s**Command and ListTask**s**Command and also to be clearer

I think we just missed the **s** when typing

File: src/main/java/seedu/duke/commands/OpenLessonLinkCommand.java [hunk 2]
```diff
@@ -16,62 +16,59 @@
...
+import static seedu.duke.common.Messages.MESSAGE_LESSON_TO_OPEN_LINK;
+import static seedu.duke.common.Messages.MESSAGE_OPENED_LESSON_LINK;
+import static seedu.duke.common.Messages.MESSAGE_UNABLE_TO_OPEN_LINK;
 
 public class OpenLessonLinkCommand extends Command {
-    public static final String MESSAGE_OPEN_LESSON_LINK = "Which lesson’s link would you like to open?";
-    public static final String OPEN_LESSON_LINK_FORMAT = "Opening %s link in browser." + System.lineSeparator();
-    public static final String MESSAGE_CANNOT_OPEN_LESSON_LINK = "Cannot open lesson link" + System.lineSeparator();
     public static final String LINUX_COMMAND = "xdg-open ";
     private final Scanner commandLineReader = new Scanner(System.in);
```
Comment (aliciatay-zls) [inline src/main/java/seedu/duke/commands/OpenLessonLinkCommand.java:25]: Maybe this line and line 37 could also make use of ui.readCommand()?

File: src/main/java/seedu/duke/common/Messages.java [hunk 3]
```diff
@@ -2,56 +2,110 @@
...
+    public static final String MESSAGE_TASKS_TO_DELETE = "Which tasks would you like to delete?";
+    public static final String MESSAGE_TASKS_TO_LIST = "Tasks for %s:";
+    public static final String MESSAGE_TASKS_TO_MARK = "Which undone tasks have you completed?";
     public static final String MESSAGE_TASKS_TO_UNMARK = "Which done tasks would you like to undo?";
-    public static final String COMMAND_VERB_UNMARK = "mark as undone";
+    public static final String MESSAGE_TASK_SET_GRADED = "Is this task graded? (Y / N)";
+    public static final String MESSAGE_TASK_SET_GRADED_INFO = "Please enter \"Y\" or \"N\"";
+    public static final String MESSAGE_TASK_SELECT_INFO = NEWLINE
+            + "Please enter the indices of the tasks you would like to %s." + NEWLINE
+            + "Separate indices with a blank space." + NEWLINE;
```
Comment (aliciatay-zls) [inline src/main/java/seedu/duke/common/Messages.java:74]: Maybe can consider:
MESSAGE_TASK_CHECK_GRADED
MESSAGE_TASK_CHECK_GRADED_INFO
MESSAGE_SELECT_TASK_INFO

**Review by aliciatay-zls — APPROVED** [review]
I'm good with this! Some very minor edits suggested.

File: src/main/java/seedu/duke/Duke.java [hunk 4]
```diff
@@ -21,8 +21,6 @@ public static void main(String[] args) {
         //Receive user input
         run();
 
-        //Exit
-        exit();
     }
 
     private static void start() {
```
[the remark this answers is not in the document]
Comment (8kdesign) [reply src/main/java/seedu/duke/Duke.java:26]: Ok replaced.

File: src/main/java/seedu/duke/commands/OpenLessonLinkCommand.java (hunk 2 above)
Comment (8kdesign) [reply src/main/java/seedu/duke/commands/OpenLessonLinkCommand.java:25]: Ok. Will replace all of it.

**Pull request merged**